buckwheat/tokenizer.py

```diff
@@ -505,6 +569,23 @@ def get_classes_from_file(file: str, lang: str, identifiers_verbose: bool = Fals
             yield obj
 
 
+# TODO: Fix extraction of Swift comments
+def get_comments_from_file(file: str) -> List[str]:
+    """
+    Given a file. Extract comments from this file.
+    :param file: the path to file.
+    :return: list of comments.
+    """
+    lang = recognize_language_file(file)['language']
+
+    if lang in TreeSitterParser.COMMENTS.keys():
```

Comment:
you can simplify it a bit 
```suggestion
    if lang in TreeSitterParser.COMMENTS:
```